src/libraries/System.Security.Cryptography.Xml/ref/System.Security.Cryptography.Xml.cs

```diff
@@ -60,8 +62,9 @@ public sealed partial class EncryptedData : System.Security.Cryptography.Xml.Enc
     public sealed partial class EncryptedKey : System.Security.Cryptography.Xml.EncryptedType
     {
         public EncryptedKey() { }
-        public string CarriedKeyName { get { throw null; } set { } }
-        public string Recipient { get { throw null; } set { } }
+        public string? CarriedKeyName { get { throw null; } set { } }
+        [System.Diagnostics.CodeAnalysis.AllowNullAttribute]
+        public string? Recipient { get { throw null; } set { } }
```

Comment:
show this return non-null? if not I'm not sure I understand AllowNull here

Reply:
This was the result of earlier PR feedback

Reply:
This combination doesn't make sense.  The AllowNull only makes sense if the property type is `string` rather than `string?`.

Reply:
Missed this one. Changed to be non-null like the implementation.